lib/Doctrine/ORM/Mapping/ClassMetadataFactory.php

```diff
@@ -74,18 +75,6 @@ class ClassMetadataFactory extends AbstractClassMetadataFactory
      */
     private $embeddablesActiveNesting = [];
 
-    /**
-     * {@inheritDoc}
-     */
-    protected function loadMetadata($name)
```

Comment:
The removal of this method can break code that extends `ClassMetadataFactory` and overrides `loadMetadata()`.